backend/calc.py

```diff
@@ -164,12 +234,21 @@ def crit_buffs(cls):
     def dh_buffs(cls):
         return {cls.BV, cls.BARD_DH, cls.DEVILMENT}
 
-    def avg_buff_effect(self):
-        return self.multiplier * self.duration / self.cd
+    def avg_buff_effect(self) -> float:
+        """Gives the dps expected from a buff"""
```

Comment:
change this docstring:

Calculates effective multiplier that a raid buff contributes to DPS. This treats a raid buff as having its effect averaged over the course of a fight and applies that average effect to all damage.

Reply:
added